src/common/iconbndl.cpp

```diff
@@ -192,6 +206,75 @@ void wxIconBundle::AddIcon(wxInputStream& stream, wxBitmapType type)
 
 #endif // wxUSE_STREAMS && wxUSE_IMAGE
 
+
+#ifdef __WINDOWS__
+
+namespace {
+
+// struct declarations taken from https://msdn.microsoft.com/en-us/library/ms997538.aspx
```

Comment:
Hmm, we actually already have very similar (but _not_ completely identical) declarations in `src/common/imagbmp.cpp` (see `wxICOHandler` code). I wonder if it would be possible to reuse them, e.g. by moving them into `include/wx/private/icondir.h` or something like this?